hack/ci/test.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export PATH=$PATH:/usr/local/go/bin/
+which kubectl
+which oc
+echo "checking nodes for cluster"
+# in CI, this cmd fails unless you provide a ns
+oc -n default get nodes
+echo "checking configuration"
+env | grep KUBE
+kubectl config view
+export DOCKER_PREFIX='dhiller'
+export DOCKER_TAG="latest"
+export KUBEVIRT_PROVIDER=external
+export GIMME_GO_VERSION=1.12.8
+export GOPATH="/go"
+export GOBIN="/usr/bin"
+source /etc/profile.d/gimme.sh
+echo "checking configuration location"
+echo "KUBECONFIG: ${KUBECONFIG}"
+# enable nested-virt
+oc get machineset -n openshift-machine-api -o json >/tmp/machinesets.json
+MACHINE_IMAGE=$(jq .items[0].spec.template.spec.providerSpec.value.disks[0].image /tmp/machinesets.json)
+NESTED_VIRT_IMAGE="sotest-rhcos-nested-virt"
```

Comment:
No expert here, but I expected that we could in openshift-CI pre-select this and directly get the right cluster. Can't that be solved in https://github.com/openshift/release/pull/6932?

Btw. where is the image coming from?

Reply:
I think I remember that the template @sallyom created was for another use case where only the `src` image type was required. So I figured she chose not to create another template and to enable nested virt directly instead.

Reply:
hm, that pulls in a lot of responsibility into the kubevirt/kubevirt codebase and probably increases the test time drastically. I guess we will need our own template then, or an additional parameter ...

But ok for now.

Reply:
I'll add this to the cleanup issue.

Reply:
The image currently used is [this one](https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml). In the cleanup I can move the setup code for enabling nested virt there.

EDIT: I meant I'll create a template from that and move the setup code to the copy.

Reply:
right, I tested with a stock template, rather than create a new one.  We can add a new template that does the enabling of nested virt later, though.  The idea here I _think_ is to get the tests running, then optimize after.